### PR TITLE
Fabo/541 mock server

### PR DESCRIPTION
**Pull request by faboweb**
> Thank you a lot for contributing to Cosmos Voyager!

### Issue

closes: #541

❤️ Thank you!

### CODE REVIEW

File: test/unit/specs/lcdClientMock.spec.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
+let signedTx = {
```
Comment (mappum) [inline test/unit/specs/lcdClientMock.spec.js]: 🤦‍♂️

**Comment by NodeGuy** [comment]
I would love to weigh in on this but don't feel yet that I have enough familiarity with the code and the protocol to offer anything meaningful.

**Comment by faboweb** [comment]
@mappum could you take a look? I would like to close this PR.

**Comment by jolesbi** [comment]
i've looked through this. hoping @mappum would approve.

File: app/src/renderer/connectors/node.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+'use strict'
+const RestClient = require('./lcdClient.js')
+const mockedRestClient = require('./lcdClientMock.js')
+const RpcWrapper = require('./rpcWrapper.js')
+const mockedRpcWrapper = require('./rpcWrapperMock.js')
+
```
Comment (mappum) [inline app/src/renderer/connectors/node.js]: :+1: I like how this is organized

**Comment by faboweb** [comment]
Thank you @mappum !

**Pull request merged**